src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -794,16 +794,8 @@ public void PublishRelease_interacts_similarly_with_PublishProfile_Configuration
             .Execute("/p:PublishProfile=test");
 
             publishOutput.Should().Pass();
-            var releaseAssetPath = System.IO.Path.Combine(helloWorldAsset.Path, "bin", "Release", ToolsetInfo.CurrentTargetFramework, rid, "HelloWorld.dll");
-            if (config == "Debug")
-            {
-                Assert.True(File.Exists(releaseAssetPath)); // We ignore Debug configuration and override it, IF its custom though, we dont use publishrelease.
-            }
-            else
-            {
-                Assert.False(File.Exists(releaseAssetPath)); // build will produce a debug asset, need to make sure this doesn't exist either.       
-                publishOutput.Should().HaveStdOutContaining("PublishRelease");
-            }
+            var configAssetPath = System.IO.Path.Combine(helloWorldAsset.Path, "bin", config, ToolsetInfo.CurrentTargetFramework, rid, "HelloWorld.dll");
+            Assert.True(File.Exists(configAssetPath)); // We cannot ignore Debug configuration and override it, because it's set as a globalproperty during publish.
```

Comment:
This change is the most controversial part of the entire PR - due to the way that publish profile-driven properties are passed into the 'main' build (they are passed via the `-p` option, so are treated as global properties), PublishRelease can no longer have any effect on the build. Is this ok?  I contend yes - the point of publish release was to make publish use Release mode by default, and the defaults for VS-authored publish profiles is to set configuration to `Release`.

Reply:
This seems fine to me. But I also think you could write it such that this isn't the case by removing the `Configuration` from the Publish Profile Evaluated Properties and Injecting the one discovered by the ReleasePropertyProjectLocator

Reply:
IMO the point of this is to get alignment with VS explicitly by using the subset of properties tracked in the array at the top of the file, so I think we should _want_ the breaking change.